packages/neon_framework/lib/src/storage/request_cache.dart

```diff
@@ -64,30 +66,32 @@ final class DefaultRequestCache implements RequestCache {
     final cacheDir = await getApplicationCacheDirectory();
     database = await openDatabase(
       p.join(cacheDir.path, 'cache.db'),
-      version: 2,
+      version: 3,
       onCreate: onCreate,
       onUpgrade: (db, oldVersion, newVersion) async {
-        final batch = db.batch();
-        if (oldVersion == 1) {
-          batch
-            ..execute('ALTER TABLE cache ADD COLUMN etag TEXT')
-            ..execute('ALTER TABLE cache ADD COLUMN expires INTEGER');
-        }
-        await batch.commit();
+        // We can safely drop the table as it only contains cached data.
+        // Non breaking migrations should not drop the cache. The next
+        // breaking change should remove all non breaking migrations before it.
+        await db.transaction((txn) async {
+          if (oldVersion <= 2) {
+            await txn.execute('DROP TABLE cache');
+            await onCreate(txn);
+          }
+        });
       },
     );
   }
 
   @visibleForTesting
-  static Future<void> onCreate(Database db, int version) async {
+  static Future<void> onCreate(DatabaseExecutor db, [int? version]) async {
     await db.execute('''
 CREATE TABLE "cache" (
-	"id"      INTEGER UNIQUE,
-	"key"     TEXT UNIQUE,
-	"value"   TEXT,
+	"account" TEXT NOT NULL,
+	"key"     TEXT NOT NULL,
+	"value"   BLOB NOT NULL,
```

Comment:
Shouldn't the blob thing be done separately as it has nothing to do with the account column?